### PR TITLE
Chore cleanup codebase

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### Description Of Task To Be Completed
Cleanup the codebase and restructure the directories.

#### Any Background Context You Want To Provide?
N/A

#### Screenshot(s)
N/A

#### What are the relevant github issues?
N/A

This PR might break a lot of things but we'll resolve as soon as we get the Trello Board or Github issues working and get started with completing the application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ MEMBERS:
...
- 
 
-As our company grows and the need for staff bonding arises, Andela Socials comes to the rescue. 
+
+As our company grows and the need for staff bonding arises, Andela Socials comes to the rescue.
 
 Andela Socials is a web app integrated into a slack channel, the app displays  a list of staff bonding activities which allows staff who share the same social interest to hangout, from lunch dates to swimming meetups, as it would improve communication, motivate the employees and increase creativity.
 
-Wanna go for swimming or to the movies next weekend? Just log on to Andela Socials  and join any social activity that interests you. When an event is created everyone who joined the social group for that event would be notified via dms on slack.
+Wanna go for swimming or to the movies next weekend? Just log on to Andela Socials  and join any social activity that interests you. When an event is created everyone who joined the social group for that event would be notified via dm's on slack.
```
Comment (ignatiusukwuoma) [inline README.md]: via DMs

File: api/models.py [hunk 2]
```diff
@@ -128,14 +128,14 @@ class Event(BaseInfo):
 
     title = models.CharField(max_length=100)
     description = models.TextField()
-    venue = models.TextField(default='Pekas Night Club')
+    venue = models.TextField()
     date = models.CharField(default='September 10, 2017', max_length=200)
```
Comment (ignatiusukwuoma) [inline api/models.py:132]: Why the default date and time?

Comment (BolajiOlajide) [reply api/models.py:132]: I remember we had an issue with the way the date was coming from the backend then. I left this as a precautionary measure incase we didn't fix it. @andela-iukwuoma

File: README.md (hunk 1 above)
Comment (BolajiOlajide) [reply README.md]: Corrected!

**Review by andela-sjames — APPROVED** [review]
Looks good to me.

**Comment by BolajiOlajide** [comment]
Hey! @andela-iukwuoma & @innocentamadi Kindly approve this PR or request changes if you feel any needs to be made.

**Review by ignatiusukwuoma — APPROVED** [review]
Nice job

**Pull request merged**